grammar.pegjs

```diff
@@ -14,6 +14,9 @@
       }
     });
   }
+
+  // https://github.com/estools/esquery/issues/68
+  input = input.replaceAll("\\/", "\\\\x2F");
```

Comment:
TODO: This is going to need to be more intelligent. 69dd283 shows a couple of failing test cases where this accidentally replaces non-regex slashes.